src/main/resources/bidder-config/brightroll.yaml

```diff
@@ -1,23 +1,56 @@
 adapters:
   brightroll:
     enabled: false
-    endpoint: http://east-bid.ybp.yahoo.com/bid/appnexuspbs
```

Comment:
On what grounds this and other changes in this configuration have been made (except new `accounts` field)?

Reply:
https://github.com/rubicon-project/prebid-server-java/pull/639 
To copy this reverted changes